docs/source/nodes_and_pipelines/nodes.md

```diff
@@ -87,6 +88,9 @@ A syntax describes function inputs and outputs. This syntax allows different Pyt
 
 Any combinations of the above are possible, except nodes of the form `node(f, None, None)` (at least a single input or output must be provided).
 
+## `*args` node functions
+It is common to have functions that take an arbitrary number of inputs. For example, a function that combines multiple dataframes. You can use the *args argument in the node function, while simply declaring the names of the datasets in the node's inputs.
```

Comment:
```suggestion
It is common to have functions that take an arbitrary number of inputs, like a function that combines multiple dataframes. You can use the `*args` argument in the node function, while simply declaring the names of the datasets in the node's inputs.
```
Just a sentence fragment.

Also, does it make sense to call them by a name, like variadic arguments, or too confusing?

Reply:
I am not sure, variadic isn't a name familiar to me. I did a quick search and seems like Python ecosystem also don't really have a name for it. i.e. `varying number of arguments.` , `non-keyworded variable length argument list`. Varidiac does show up in docsbut in the standard doc they call the section Arbitary argument list" " https://docs.python.org/3/tutorial/controlflow.html#arbitrary-argument-lists"